rel-eng/releasers.conf

```diff
@@ -1,32 +1,11 @@
 [fedora]
 releaser = tito.release.FedoraGitReleaser
-branches = main f36 f35
+branches = main f39 f40
 
-[centos-9-stream]
+[centos-10-stream]
 releaser = tito.release.CentosGitReleaser
-branches = c9s
-required_bz_flags = release+
+branches = c10s
 
-[rhel-9.0]
+[rhel-10]
 releaser = tito.release.DistGitReleaser
-branches = rhel-9.0.0
-required_bz_flags = release+
-# Change this if you wish to use a placeholder "rebase" bug if none
-# are found in the changelog.
-placeholder_bz =
-
-[rhel-9.1]
-releaser = tito.release.DistGitReleaser
-branches = rhel-9.1.0
-required_bz_flags = release+
-# Change this if you wish to use a placeholder "rebase" bug if none
-# are found in the changelog.
-placeholder_bz =
-
-[rhel-9.2]
-releaser = tito.release.DistGitReleaser
-branches = rhel-9.2.0
-required_bz_flags = release+
-# Change this if you wish to use a placeholder "rebase" bug if none
-# are found in the changelog.
-placeholder_bz =
+branches = rhel-10-main
```

Comment:
note that this will not actually used in practice, since updating the downstream `rhel-10-main` will always go through c10s